src/main/java/picard/util/IntervalListTools.java

```diff
@@ -237,7 +237,8 @@
 
     @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME,
             doc = "One or more interval lists. If multiple interval lists are provided the output is the" +
-                    "result of merging the inputs. Supported formats are interval_list and VCF.", minElements = 1)
+                    "result of merging the inputs. Supported formats are interval_list and VCF." +
+                    "If file extension is unrecognized, assumes file is interval_list", minElements = 1)
```

Comment:
How about specifically noting that for standard input one should write `/dev/stdin` as the input file?